base/Dockerfile.rhel

```diff
@@ -18,5 +18,9 @@ RUN INSTALL_PKGS=" \
     yum install -y --setopt=tsflags=nodocs --setopt=install_weak_deps=False ${INSTALL_PKGS} && \
     ( test -e /usr/bin/python ||  alternatives --set python /usr/bin/python3 ) && \
     yum clean all && rm -rf /var/cache/*
+
+# Enable x509 common name matching for golang 1.15 and beyond.
+ENV GODEBUG=${GODEBUG}:x509ignoreCN=0
```

Comment:
does this : logic work? `${GODEBUG}` is most probably empty.

Can we grep over all Dockerfiles and repos in general whether components set GODEBUG today? Could be deep in some operator go code.

Reply:
only one I found in my local checkouts:
```
RUN GODEBUG=tls13=1 go build ./cmd/cluster-openshift-apiserver-operator
```
@vareti can you make our operator compatible with this change, i.e. use something similar as here to concatenate GODEBUG env var values?

Reply:
I did a search in our github org and only found router test images setting GODEBUG, but some of those indexes were as old as July.

Reply:
@sttts @vareti I clone master for all repos we ship in the most recently nightly, and coao and svcat were the only repos that set GODEBUG and only for build. That won't be impacted by the change proposed here since it sets an absolute value that won't change if GODEBUG is set in the environment. Anyway, the flag in question has no effect on builds, it's runtime only.

Reply:
@marun thanks for looking it up and making sure it does not effect us.